File: pset6/cash/cash.py
from cs50 import get_float

change = 0
coins = 0

while True:
    change = int(get_float("Change owned: ") * 100)
    if change > 0:
        break

while change > 0:
    if change >= 25:
        change -= 25
        coins += 1

    elif change >= 10:
        change -= 10
        coins += 1

    elif change >= 5:
        change -= 5
        coins += 1

    elif change >= 1:
        change -= 1
        coins += 1

print(coins)

File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float n;
    do
    {
        n = get_float("Change owed: ");
    }
    while (n < 0);

    int cents = round(n * 100);
    int coins = 0;

    while (cents > 0)
    {
        if (cents >= 25)
        {
            cents -= 25;
            coins++;
        }
        else if (cents >= 10)
        {
            cents -= 10;
            coins++;
        }
        else if (cents >= 5)
        {
            cents -= 5;
            coins++;
        }
        else if (cents >= 1)
        {
            cents -= 1;
            coins++;
        }
    }
    printf("%i\n", coins);
}